src/VirtualClient/VirtualClient.Actions.UnitTests/Examples/Wrathmark/TestWrathmark.txt

```diff
@@ -0,0 +1,1070 @@
+				6 B B W W B B B B 
+				5 B W B W B B B B 
+				4 B B W B W W B B 
+				3 B B B W B B W - 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W19 B=31
+Seeking to end of board
+
+14: g2=32  a2<32  h3<32  b7=38  g7<38  g8<38  
+
+Evaluated 723,741 boards in 0:00.088 8,224,329 boards/sec  Total time used: 6:02.181
+best move for W is at b7 (score 38)
+				8 - - B - - B - - 
+				7 B W W W W B - B 
+				6 B W W W B B B B 
+				5 B W B W B B B B 
+				4 B B W B W W B B 
+				3 B B B W B B W - 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W21 B=30
+Seeking to end of board
+
+14: g1=24  g2<24  h3=26  e8<26  d8<26  b8<26  a8<26  
+
+Evaluated 728,909 boards in 0:00.068 10,719,250 boards/sec  Total time used: 6:02.249
+best move for B is at h3 (score 26)
+				8 - - B - - B - - 
+				7 B W W W W B - B 
+				6 B W W W B B B B 
+				5 B W B W B B B B 
+				4 B B W B W W B B 
+				3 B B B W B B B B 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W20 B=32
+Seeking to end of board
+
+12: g2=19  a2<19  g7<19  h8=38  g8<38  
+
+Evaluated 83,126 boards in 0:00.010 8,312,600 boards/sec  Total time used: 6:02.259
+best move for W is at h8 (score 38)
+				8 - - B - - B - W 
+				7 B W W W W B - W 
+				6 B W W W B B B W 
+				5 B W B W B B B W 
+				4 B B W B W W B W 
+				3 B B B W B B B W 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W26 B=27
+Seeking to end of board
+
+12: g1=23  g2<23  e8=25  d8<25  b8<25  a8=26  
+
+Evaluated 81,221 boards in 0:00.007 11,603,000 boards/sec  Total time used: 6:02.266
+best move for B is at a8 (score 26)
+				8 B - B - - B - W 
+				7 B B W W W B - W 
+				6 B W B W B B B W 
+				5 B W B B B B B W 
+				4 B B W B B W B W 
+				3 B B B W B B B W 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W22 B=32
+Seeking to end of board
+
+10: a2=23  g7=32  g8=36  e8<36  b8=38  
+
+Evaluated 20,089 boards in 0:00.002 10,044,500 boards/sec  Total time used: 6:02.268
+best move for W is at b8 (score 38)
+				8 B W B - - B - W 
+				7 B W W W W B - W 
+				6 B W B W B B B W 
+				5 B W B B B B B W 
+				4 B B W B B W B W 
+				3 B B B W B B B W 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W24 B=31
+Seeking to end of board
+
+10: g1=23  g2<23  e8<23  d8=26  
+
+Evaluated 7,725 boards in 0:00.000 -2,147,483,648 boards/sec  Total time used: 6:02.268
+best move for B is at d8 (score 26)
+				8 B W B B - B - W 
+				7 B W B B B B - W 
+				6 B B B B B B B W 
+				5 B W B B B B B W 
+				4 B B W B B W B W 
+				3 B B B W B B B W 
+				2 - B B B B W - W 
+				1 - W W W W W - - 
+				  a b c d e f g h 
+						    Score: W19 B=37
+Seeking to end of board
+
+ 8: g2=30  a2<30  g7=38  g8<38  e8<38  
+
+Evaluated 1,854 boards in 0:00.000 -2,147,483,648 boards/sec  Total time used: 6:02.268
```

Comment:
Just curious, how come these scores are exactly the same cross tests?

Reply:
The algorithm is completely deterministic.  It will evaluate exactly the same boards in all (correct) implementations and come to the same conclusion.  However there seems to be an overflow if the time is zero.